airbyte/destinations/base.py

```
@@ -48,6 +48,7 @@ def __init__(
         executor: Executor,
         name: str,
         config: dict[str, Any] | None = None,
+        config_change_callback: Callable[[dict[str, Any]], None] | None = None,
```

Ruff

Undefined name `Callable`

(F821)